app/Main.hs

```diff
@@ -55,107 +54,104 @@ readConfig = do
 main :: IO ()
 main = do
```

Comment:
We should make sure paths appear qualified in both success and error messages.

```
coffer copy 'b1#/a' 'b2#/a'  
[SUCCESS] Copied '/a/b' to '/a/b'.
```

Reply:
I think we should create a separate issue for this because after fixing this many `bats` tests would fail. And it would be more comfortable to write tests if we merge #18 and #21.

Reply:
Sure, can you please create an issue for that?

Reply:
Created #52

Reply:
Looks great, thanks!